apps/web/playwright/reschedule.e2e.ts

```diff
@@ -50,11 +56,11 @@ test.describe("Reschedule Tests", async () => {
     const user = await users.create();
     // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
     const booking = await bookings.create(user.id, user.username, user.eventTypes[0].id!, {
-      status: BookingStatus.CANCELLED,
+      status: BookingStatus.ACCEPTED,
```

Comment:
CANCELLED booking can't be rescheduled through `reschedule/[uid]`. So, I changed it.